cucumber.conf.js

```diff
@@ -0,0 +1,35 @@
+const {
+  Before,
+  BeforeAll,
+  AfterAll,
+  After,
+  setDefaultTimeout,
+} = require("@cucumber/cucumber");
+const { chromium } = require("playwright");
+const { config } = require("./config");
+const {
+  deleteFolder,
+} = require("./tests/cucumber/acceptance/helpers/HttpHelpers");
```

Comment:
Just asking if you want to have `tests/cucumber/acceptance` structure

Reply:
IMO ./tests/acceptance/helpers/HttpHelpers